src/java.net.http/share/classes/java/net/http/HttpRequest.java

```diff
@@ -561,7 +567,8 @@ public interface BodyPublisher extends Flow.Publisher<ByteBuffer> {
      * convert common high-level Java objects into a flow of data suitable for
      * sending as a request body:
      *
-     *  <pre>{@code    // Request body from a String
+     * {@snippet :
+     *   // Request body from a String
```

Comment:
Since this snippet contains three different individual example it would be better to have each example in its own snippet.